addons/crm/tests/test_crm_pls.py

```diff
@@ -399,3 +399,17 @@ def test_settings_pls_start_date(self):
         res_config_new = resConfig.new()
         self.assertEqual(fields.Date.to_string(res_config_new.predictive_lead_scoring_start_date),
             str_date_8_days_ago, "If config param is not a valid date, date in settings should be set to 8 days before today")
+
+    def test_stages_team_specific(self):
+        Lead = self.env['crm.lead']
+        state_value = 'correct'
+        source_id = self.env['utm.source'].search([], limit=1).id
+        state_id = self.env['res.country.state'].search([], limit=1).id
+        country_id = self.env['res.country'].search([], limit=1).id
+        team_id = self.env['crm.team'].create([{'name': 'Team Test'}]).id
+        self.env['crm.stage'].search([('team_id', '=', False)]).write({'team_id': team_id})
+        stage_id = self.env['crm.stage'].search([], limit=1).id
+        Lead.create(
+            self._get_lead_values(team_id, 'team', country_id, state_id, state_value, state_value, source_id, stage_id)
+        )
+        Lead._cron_update_automated_probabilities()
```

Comment:
Could be great to add a small docstring to the test explaining its purpose, as it is not obvious when looking at it (ok commit message is clear, but a small docstring here would be welcomed)